src/amp-story-player/amp-story-player-impl.js

```diff
@@ -352,7 +356,11 @@ export class AmpStoryPlayer {
         story,
         iframe,
         // In case it is the first story, it becomes immediately visibile
-        idx === 0 ? VisibilityState.VISIBLE : VisibilityState.PRERENDER
+        idx === 0 ? VisibilityState.VISIBLE : VisibilityState.PRERENDER,
+        // Autoplaying is applied even if the first story is added dynamically
+        idx === 0 && this.isAutoplaying_
```

Comment:
WDYT of making `this.isAutoplaying_ = this.element_.getAttribute('autoplay') && idx === 0` so we don't have to pass an extra argument in all these methods, and just check `this.isAutoplaying_` when it's needed in `this.getEncodedLocation`. 

It's good for code readability to keep the number of arguments in methods low.

Reply:
How about we change `this.isAutoplaying_` from a field to a getter property? I think reading directly the autoplay attribute every time from DOM would require a more robust solution (in the case it's changed programatically), but as it is right now (only making the first story go on preview mode), I think it could work.

I would also suggest `this.getEncodedLocation_` stay as a somewhat pure function, but if that's not the intention, I could put this autoplay embed mode logic there.

Reply:
Leaving like it is is fine, we can do a cleanup later if needed.